contracts/anchor/Anchor.sol

```diff
@@ -19,119 +19,56 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./AnchorI.sol";
 
 import "../consensus/ConsensusModule.sol";
-import "../lib/CircularBufferUint.sol";
-import "../lib/MerklePatriciaProof.sol";
-import "../lib/RLP.sol";
 import "../proxies/MasterCopyNonUpgradable.sol";
+import "../anchor/StateRootProvider.sol";
```

Comment:
I would propose as a good practice to sort includes alphabetically. This easies the merge conflicts in future.

Reply:
Sorted by file path now.